entrypoint.py

```diff
@@ -115,7 +119,10 @@ def annotate_pr(message: str) -> None:
     has_failure: bool = False
     for f in files:
         result: ValidationResult = validate(
-            f, case_version=case_version, abort_on_first=abort_on_failure
+            f,
+            case_version=case_version,
+            abort_on_first=abort_on_failure,
+            allow_warnings=allow_unrecognized,
```

Comment:
This line shows the current granularity potential of this configuration parameter.

`case_validate.validate` does not have a specific option for ignoring warnings generated by the IRI typo-checker function.  It only has an option for ignoring _all_ warnings, which also includes SHACL validation warnings.